### DISCUSSION TITLE
No more standalone binaries upload in release since 1.5.8

### DISCUSSION THREAD

**Discussion by Klaitos** [swc-project/swc#9034] [Q&A]
Hello,
We were downloading swc binaries from Github in our CI to generate source maps. 
Since v1.5.8, only .node files are uploaded and not executable on system anymore. Is this an expected behavior ?

Assets uploaded before : 
- swc-darwin-arm64

Assets now : 
- swc.darwin-arm64.node (little change in naming too after swc)

Thanks

**Comment by kdy1** [comment]
No. It's not expected. I think I need to adjust file paths in https://github.com/swc-project/swc/blob/328cb4b248c23e8c3a37ac12fdf67b19b1177d9e/scripts/cli_upload_gh_release.sh

Can you file an issue?

**Reply by kdy1** [reply]
I refactored the structure of the main repository to make it _correct_ monorepo but it had some side effects
